client/tinychain/app.py

```diff
@@ -78,6 +80,18 @@ def __ns__(self, _context, _name_hint):
     def __ref__(self, name):
         return ModelRef(self, name)
 
+    @classmethod
+    def key(cls):
+        """A Column object which will be used as the key for a given model."""
+        return [Column(cls.class_name() + "_id", U32)]
+
+    @classmethod
+    def class_name(cls):
```

Comment:
This might work better as a helper function, then it could be called on any class or instance (instead of `x.__class__.__name__`, for example)